lib/oli/publishing/part_mapping_refresh_adapter.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Oli.Publishing.PartMappingRefreshAdapter do
+  alias Oli.Publishing.Publication
+
+  @moduledoc """
+    Behaviour for spawning a function that freshes the part_mapping materialized view:
+  """
+
+  @doc """
+  Defines a callback to be used by refresh adapters
+  """
+  @callback maybe_refresh_part_mapping({:ok, %Publication{}} | {:error, %Ecto.Changeset{}}) ::
```

Comment:
NIT: maybe we can declare some type with `@type` to avoid writing this all the times, wdyt?

Reply:
Gotcha. Let me know if there's any other way to define and import `@types` from what I did.